### DISCUSSION TITLE
Is there a way to change the cursor size?

### DISCUSSION THREAD

**Discussion by vdegenne** [WayfireWM/wayfire#2428] [Q&A] [answered]
Is there a way to change the cursor size? I often lose track of it, especially because I am mostly using my keyboard and I have two screens, I'd like to find a way to make the pointer bigger so my eyes can spot it with more ease.
Changing the color would be great too.

Looking forward for your help, Thanks.

**Comment by vdegenne** [comment]
I found a way to change the cursor size using `gnome-control-center` and going in "accessibility", **HOWEVER** the large cursor only works in gnome related applications... For instance moving my cursor on an `alacritty` terminal window will make the cursor go back to small size...

**Comment by soreau** [comment, marked as answer]
See the wiki [here](https://github.com/WayfireWM/wayfire/wiki/Configuration#inputcursor_size) and [here](https://github.com/WayfireWM/wayfire/wiki/Configuration#inputcursor_theme).

**Reply by vdegenne** [reply]
Thanks! It doesn't change the cursor over Gnome apps though so I kept the gnome accessibility value on too.
